Sources/EpoxyBars/BarInstaller/BarStackView.swift

```diff
@@ -4,32 +4,34 @@
 import EpoxyCore
 import UIKit
 
+// MARK: - BarStackView
+
 /// A stack of arbitrary bar views, typically fixed to either the top or bottom of a view
 /// controller.
```

Comment:
Can we update this doc comment to indicate that this can be used as an arbitrary stack of views as well?